Demo/ColorSliderDemo.xcodeproj/project.pbxproj

```diff
@@ -326,7 +326,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 9BZNV2TE99;
+				DEVELOPMENT_TEAM = 6J9MZ7E65M;
```

Comment:
Please revert these changes (or remove the `DEVELOPMENT_TEAM` setting from the pbxproj altogether).

Reply:
fixed